encode.go

```diff
@@ -623,19 +626,18 @@ func (e *Encoder) encodeStruct(ctx context.Context, value reflect.Value, column
 			// omit encoding
 			continue
 		}
-		value, err := e.encodeValue(ctx, fieldValue, column)
+		ve := e
```

Comment:
Please explain why this new process is necessary 🙏

Reply:
ping @igorpeshansky

Reply:
Apologies, I completely missed this. This is to ensure that the value is encoded with flow style even f the base encoder doesn't set it, without modifying the base encoder. Should I add a comment to that effect?

Reply:
I see ! Thank you for the reply.